Sources/SwifterSwift/UIKit/UIScrollViewExtensions.swift

```diff
@@ -32,6 +32,99 @@ public extension UIScrollView {
         return UIGraphicsGetImageFromCurrentImageContext()
     }
 
+    /// SwifterSwift: The currently visible region of the scroll view.
+    var visibleRect: CGRect {
+        let contentWidth = contentSize.width - contentOffset.x
+        let contentHeight = contentSize.height - contentOffset.y
+        return CGRect(origin: contentOffset,
+                      size: CGSize(width: min(min(bounds.size.width, contentSize.width), contentWidth),
+                                   height: min(min(bounds.size.height, contentSize.height), contentHeight)))
+    }
+
+}
+
+extension UIScrollView {
+
+    /// SwifterSwift: Scroll to the top-most content offset.
+    /// - Parameter animated: `true` to animate the transition at a constant velocity to the new offset, `false` to make the transition immediate.
+    func scrollToTop(animated: Bool = true) {
+        setContentOffset(CGPoint(x: contentOffset.x, y: 0), animated: animated)
+    }
+
+    /// SwifterSwift: Scroll to the left-most content offset.
+    /// - Parameter animated: `true` to animate the transition at a constant velocity to the new offset, `false` to make the transition immediate.
+    func scrollToLeft(animated: Bool = true) {
+        setContentOffset(CGPoint(x: 0, y: contentOffset.y), animated: animated)
+    }
+
+    /// SwifterSwift: Scroll to the bottom-most content offset.
+    /// - Parameter animated: `true` to animate the transition at a constant velocity to the new offset, `false` to make the transition immediate.
+    func scrollToBottom(animated: Bool = true) {
+        setContentOffset(CGPoint(x: contentOffset.x, y: max(0, contentSize.height - bounds.height)), animated: animated)
+    }
+
+    /// SwifterSwift: Scroll to the right-most content offset.
+    /// - Parameter animated: `true` to animate the transition at a constant velocity to the new offset, `false` to make the transition immediate.
+    func scrollToRight(animated: Bool = true) {
+        setContentOffset(CGPoint(x: max(0, contentSize.width - bounds.width), y: contentOffset.y), animated: animated)
+    }
+
+    /// SwifterSwift: Scroll up one length of the scroll view.
```

Comment:
What does "one length" means? Wouldn't be better to describe this as `page` since seems to be the same thing?